tests/zep/config.yaml

```diff
@@ -0,0 +1,22 @@
+datasources:
+  my_pg_ds:
+    type: postgres
+    name: my_pg_ds
+    connection_str: "postgres://foo.bar"
+    assets:
+      my_table_asset_wo_splitters:
+        type: table
+        name: my_table_asset_wo_splitters
+        table_name: my_table
+      with_splitters:
+        type: table
+        name: with_splitters
+        table_name: another_table
+        column_splitter:
+              method_name: foobar_it
+              column_name: my_column
+              name: my_splitter
+              template_params:
+                - alpha
```

Comment:
`template_params` is now `param_defaults` and is a dict. This is in flux though and we can update it when other changes go in.

Reply:
Just updated.